fixed issue 404 (#580)
mautic/user-documentation: docs/contacts/contacts_overview.rst

By default Mautic uses the **list view**, but you can also choose to switch to the **card view** also known as **grid view**, which uses avatars to depict the Contacts visually using cards. 

> Click the **V** button to switch between the list and card views.

Reviewer: I believe, for keyboards, we don't say "click" and "button". Let's reword this:

```suggestion
Press **V** on your keyboard to switch between the list and card views.
```

Author: Alrighty, I've made the tweaks.
